src/wallet.js

```diff
@@ -353,8 +353,14 @@ MyWallet.initializeWallet = function (pw, decryptSuccess, buildHdSuccess) {
     };
     return MyWallet.wallet.loadExternal.bind(MyWallet.wallet)().catch(loadExternalFailed);
   };
-
-  return Promise.resolve().then(doInitialize).then(tryLoadExternal);
+  var incStats = function () {
+    var gh = MyWallet.wallet.getHistory.bind(MyWallet.wallet);
```

Comment:
Yeah I though about this... The problem is the getHistory calls on mywallet v3 are used ignoring the returned promise. So, I cannot use them without some big refactoring in the login process and I don't want to split this collection data features into my-wallet-v3 and frontend...